src/Utilities/UnitTests/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
     "xunit": "2.1.0",
+    "xunit.netcore.extensions": "1.0.0-prerelease-00116",
```

Comment:
Is the `netstandard1.3` for .netcore builds? AFAICS, only `xunit.netcore.extensions` seems to have the `PlatformSpecific` attribute that we need here, even for !netcore builds . I could be wrong here!